site/en/docs/privacy-sandbox/overview/index.md

```diff
@@ -1,180 +1,137 @@
+* [**First-Party Sets**](/docs/privacy-sandbox/first-party-sets): Allow related
+  domain names owned by the same entity to declare themselves as belonging to
+  the same first party.
+* [**Shared Storage**](/docs/privacy-sandbox/shared-storage/): Create a
+  general-purpose API which allows sites to store and access unpartitioned
+  cross-site data. This data must be read in a secure environment to prevent leakage.
+* [**SameSite cookies**](https://web.dev/samesite-cookies-explained/): Secure
+  sites by explicitly marking cross-site cookies.
+* [**Storage Partitioning**](https://github.com/privacycg/storage-partitioning):
+  Enable all forms of [user agent state](https://github.com/privacycg/storage-partitioning#user-agent-state),
+  such as `localStorage` or cookies, to be double-keyed: by the top-level site
+  as well as the origin of the resource being loaded, rather than a single
+  origin or site.
+* [**Fenced Frames**](/docs/privacy-sandbox/fenced-frame): Securely embed content onto a page without sharing cross-site data.
+* [**Network State Partitioning**](https://github.com/MattMenke2/Explainer---Partition-Network-State):
+  Prevent browser network resources being shared across first-party contexts, by ensuring that every request has a network partition key that must match in order for resources to be reused.
+* [**HTTP Cache Partitioning**](/blog/http-cache-partitioning): Improve
+  security and privacy by partitioning the browser HTTP cache.
+* [**Federated Credential Management (FedCM)**](/docs/privacy-sandbox/fedcm/): Support federated identity without sharing the user's email address or other identifying information with a third-party service or website, unless the user explicitly agrees to do so.
 
 ### Show relevant content and ads
 
-* [**Topics API**](/docs/privacy-sandbox/topics): Enable interest-based advertising. Designed so
-  that it doesn't require third-party cookies and cannot be used by third parties to track user
-  browsing behavior across sites. The Topics API proposes a mechanism to map website hostnames to
-  topics of interest, and provides a JavaScript API that returns coarse-grained topics a user
-  might currently be interested in, based on their recent browsing activity.
-* [**FLEDGE**](/docs/privacy-sandbox/fledge): Ad selection to serve remarketing and custom audience
-use cases, designed so that it cannot be used by third parties to track user browsing behavior across
-sites.  FLEDGE is the first experiment to be implemented in Chromium within the
-[TURTLEDOVE](https://github.com/WICG/turtledove) family of proposals.
-
+* [**Topics API**](/docs/privacy-sandbox/topics): Enable interest-based
+  advertising without use of third-party cookies or tracking user behavior
+  across sites.
+* [**FLEDGE**](/docs/privacy-sandbox/fledge): Ad selection to serve remarketing
+  and custom audience use cases, designed so that it cannot be used by third
+  parties to track user browsing behavior across sites.
```

Comment:
```suggestion
* [**FLEDGE**](/docs/privacy-sandbox/fledge): Ad selection to serve remarketing
  and custom audience use cases, designed so that it cannot be used by third
  parties to track user browsing behavior across sites. FLEDGE is the first
  experiment to be implemented in Chromium from the
  [TURTLEDOVE](https://github.com/WICG/turtledove) family of proposals.
```